### ISSUE TITLE
Cities are not generated in 1.16.4

### ISSUE THREAD

**Issue by atzelpool** [McJty/LostCities#340]
I have tried the ways you mention and it has not worked for me to copy the .tom file to the defaultconfig folder

**Comment by McJty** [comment]
A bit more info? Do you mean on a server?

**Comment by atzelpool** [comment]
That's right, I'm on a server and I did what they recommended to copy the .toml file, could you make a video tutorial?

**Comment by atzelpool** [comment]
since I tried with the .toml file, also with the generator put the profile and the server does not generate structures

**Comment by McJty** [comment]
There is a bug currently. You need to manually edit that .toml file to replace the <CHECK> with the profile you want

**Comment by atzelpool** [comment]
Could you explain me in detail?

**Comment by McJty** [comment]
Edit that toml file using a text editor and replace the string <CHECK> with whatever profile you want (like 'default')

**Comment by atzelpool** [comment]
#General settings
[profiles]
	selectedCustomJson = ""
	selectedProfile = "rarecities"
